pymc/sampling/jax.py

```diff
@@ -669,9 +669,7 @@ def sample_jax_nuts(
     else:
         log_likelihood = None
 
-    attrs = {
-        "sampling_time": (tic2 - tic1).total_seconds(),
-    }
+    attrs = {"sampling_time": (tic2 - tic1).total_seconds(), "tuning_steps": tune}
```

Comment:
Add trailing comma so pre-commit formats this into one entry per line
```suggestion
    attrs = {"sampling_time": (tic2 - tic1).total_seconds(), "tuning_steps": tune,}
```